static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1053,6 +1053,35 @@ $quote-mark: 35px;
         }
     }
 }
+
+// *************** Opinion Showcase Styles ***************
+
+.content--type-guardianview.has-feature-showcase-element,
```

Comment:
This is quite a lot of specificity needed here. 

Given opinion has its own template, can't we just add different classes and target those instead of this amount of specificity?